packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -409,6 +428,10 @@ class FlutterErrorDetails extends Diagnosticable {
         ));
       }
     }
+    if (debugCreator != null && describeCreator != null) {
+      properties.add(DiagnosticsNode.message(''));
```

Comment:
Use
`ErrorSpacer()`
instead of
`DiagnosticsNode.message('')`
This was added since you started working on this CL.